packages/core/src/layout/Sidebar/Items.tsx

```diff
@@ -61,13 +61,27 @@ const useStyles = makeStyles<Theme>(theme => ({
 type SidebarItemProps = {
   icon: typeof SvgIcon;
   text: string;
+  avatarUrl?: string;
   to?: string;
   onClick?: () => void;
 };
 
+const LoginAvatar: FC<SidebarItemProps> = ({ icon: Icon, avatarUrl }) => {
+  if (avatarUrl !== undefined && avatarUrl !== '')
+    return (
+      <img
+        alt="userAvatarIcon"
+        src={avatarUrl}
+        style={{ borderRadius: '50%', width: '24px', height: '24px' }}
+      />
+    );
+  return <Icon fontSize="small" />;
+};
+
 export const SidebarItem: FC<SidebarItemProps> = ({
   icon: Icon,
```

Comment:
I prefer if this one instead is made to be of essentially of type `SvgIcon | Avatar` and it just gets placed the exact same way in the jsx. It's not necessary to expose the url to a generic item, since all it wants to do is to place something icon-like and some text side by side as a link.

Make use of https://material-ui.com/components/avatars/

Reply:
(Or essentially, make it a JSX.Element so the caller instantiates it)

Reply:
didn't know about it! very cool, ill change to it